src/main/kotlin/de/livepoll/api/service/PollService.kt

```diff
@@ -123,71 +158,123 @@ class PollService(
                 this.startDate = null
                 this.endDate = null
             }
```

Comment:
Sorry, I couldn't select the relevant lines, just open the file manually.

Reply:
The case that one is null and the other is not is not valid. Therefore the dates are not changed. Either the poll is scheduled or started manually. As far as I know, there are no plans for a hybrid variant

Reply:
In theory, the frontend already supports such a hybrid model. Your choice if you want to implement it ...

Reply:
> The case that one is null and the other is not is not valid. Therefore the dates are not changed. Either the poll is scheduled or started manually. As far as I know, there are no plans for a hybrid variant

But if it is not valid, we should at least send an error back to avoid inconsistent states, right? Otherwise, the frontend assumes that the `startDate` and `endDate` are updated, although we haven't changed them.

Reply:
Doesn't matter as the frontend should reload the UI after the update request.

Reply:
I also think that we can just ignore the case right now

Reply:
ok fine